### PR TITLE
Clarify situation around the UNION reordering deprecation

### PR DESCRIPTION
**Pull request by NataliaIvakina**
Supersedes #219

### CODE REVIEW

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -38,7 +38,7 @@ a|This query builds a cartesian product between disconnected patterns.
 |Category
 m|PERFORMANCE
 |GQLSTATUS code
-m|03N90
+m|1n003N90
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/notifications/all-notifications.adoc:41]:
```suggestion
m|03N90
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-224.surge.sh

Updated pages:
- [List of all server notification codes](https://neo4j-docs-status-codes-224.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-224.surge.sh

Updated pages:
- [List of all server notification codes](https://neo4j-docs-status-codes-224.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-224.surge.sh

Updated pages:
- [List of all server notification codes](https://neo4j-docs-status-codes-224.surge.sh/status-codes/5/notifications/all-notifications)

**Review by renetapopova — APPROVED** [review]

**Review by gem-neo4j — APPROVED** [review]
This seems reasonable to me :) thanks

**Pull request merged**